python/docs/source/reference/timestream/string.md

```diff
@@ -6,6 +6,9 @@
 .. autosummary::
    :toctree: ../apidocs/
 
+    Timestream.json
```

Comment:
If we can't get `json` to work as it was, we can either drop it for now, or expose the primitive `json_field` which takes the field name: `e.json_field("name")`. The support was previously relying on detecting `json(e).name` and converting it to that instruction, but that is only part of the Fenl compilation path, not what the Python stuff goes through.

I think maybe we should skip JSON for now.

Reply:
removed. issue made to do later #768